ltiauthenticator/utils.py

```diff
@@ -168,7 +168,7 @@ async def get_lms_access_token(
     params = {
         "grant_type": "client_credentials",
         "client_assertion_type": "urn:ietf:params:oauth:client-assertion-type:jwt-bearer",
-        "client_assertion": token.decode(),
+        "client_assertion": token,
```

Comment:
token was bytes, but is now a string, so we didn't need to decode the bytes into a string any more. token was set by jwt.encode earlier, and never used besides here in a way that mattered.